### DISCUSSION TITLE
Gpt4 models missing when starting webui with OPENAI_API_KEY

### DISCUSSION THREAD

**Discussion by Seviks** [open-webui/open-webui#2262] [Q&A]
I'm starting the webui with:
`docker run -d -p 3000:8080 -e OPENAI_API_KEY="secretapikey" -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

I can then select my Ollama models or gtp-3.5-turbo,  gtp-3.5-turb-0125 or  gtp-3.5-turbo-0301. 

How can I use the gpt4 models?

Some info:
- Open web ui version  v0.1.124
- Docker is running on Windows 11
- Ollama (v0.1.32) is running in the background in wsl2.
- GPT-3.5 models work
- OpenAI:  There are credits and usage limit is not reached

**Comment by justinh-rahb** [comment]
If the other `gpt-*` models are showing, just not the `gpt-4*` ones then I'd say it seems your API key is restricted.

**Reply by Seviks** [reply]
Thanks for your hint. This may be, as I'm not experienced with the openAI API. But in the API settigns it seems ok:
![grafik](https://github.com/open-webui/open-webui/assets/4014268/684f724b-485f-4ea5-a9f3-db5a829f2ca7)

Fruther hints are appreciated.

If I enter a wrong key, no model is shown.

**Reply by cheahjs** [reply]
https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4-gpt-4-turbo-and-gpt-4o

> After you have made a successful payment of $5 or more (usage tier 1), you'll be able to access the [GPT-4, GPT-4 Turbo, and GPT-4o](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo) models via the OpenAI API.

**Reply by Seviks** [reply]
I'm already on Tier 1, so this is not the solution. However, I replicated every step on my home computer (running Pop-Os) with the same version of open-webui and there everything works fine. So it seems I have to check the configuration on the windows computer on my end.
Thank you for your answers and sorry for the noise (I will add a solution if I find one).

**Reply by Seviks** [reply]
After I removed  container **and the volume** and recreated the container with the command above, all is working as expected. 
Maybe it was because for the first time I run 
`docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

but I couldn't replicate the problem
